test/test_retry.py

```diff
@@ -11,15 +13,87 @@
 from nameko.testing.services import entrypoint_waiter
 
 from nameko_amqp_retry import Backoff, BackoffPublisher
+from nameko_amqp_retry.backoff import get_backoff_queue_name
+from nameko_amqp_retry.messaging import consume
 from nameko_amqp_retry.rpc import rpc
 
 
+def retry(fn):
+    """ Barebones retry decorator
+    """
+    def wrapper(*a, **kw):
```

Comment:
`kw_args` maybe here? cause we use it elsewhere, also not sure what `a` is for :)